go.mod

```diff
@@ -23,5 +23,7 @@ require (
 	github.com/urfave/cli v1.22.1
 	github.com/vishvananda/netlink v1.1.0
 	github.com/willf/bitset v1.1.11
-	golang.org/x/sys v0.0.0-20201107080550-4d91cf3a1aaf
+	github.com/yalue/native_endian v0.0.0-20180607135909-51013b03be4f
```

Comment:
seems unmaintained?

Reply:
The package is incredibly trivial (compile-time selection of whether to use `encoding/binary.BigEndian` or `encoding/binary.LittleEndian`) -- you only need to update it when Go adds a new architecture.

Reply:
The list is outdated for gccgo, looking at the golang.org/x packages, they have two styles:

https://github.com/golang/net/blob/6772e930b67bb09bf22262c7378e7d2f67cf59d1/internal/socket/sys.go#L13

https://github.com/golang/sys/blob/4bcb84eeeb78d47aae60461dac79349e427b5349/cpu/byteorder.go#L44

Reply:
I was hoping to be able to avoid doing the checks at runtime, but okay I'll implement it using the `unsafe.Pointer` trick...

(And of course none of these `golang.org/x/` packages export this *incredibly useful* information...)